src/query/util/promqlengine/common.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 Uber Technologies, Inc.
```

Comment:
I think the conviention is to have `doc.go` if its an empty file just with docs.

Reply:
https://github.com/m3db/m3/pull/3882/commits/8711bb2342b26880cd46edc95008c80093c7d639